### ISSUE TITLE
Why does the table get inside the screen?

### ISSUE THREAD

**Issue by iman2420** [InQBarna/TableFixHeaders#132]
I used this table for the tablet. But as you can see in the screenshot below, the width of the table is fixed to the screen, which causes the width-sectional scroll to disappear.
How to fix this problem?
![image](https://user-images.githubusercontent.com/9848715/56745860-63dead00-6790-11e9-91d1-7a79aa459198.png)

**Comment by BraisGabin** [comment]
I don't get your problem... If we can show all the content there is not need to scroll, right? Am I missing something here?

**Comment by iman2420** [comment]
Your answer is absolutely correct. But when the length of the text is long, the lines should not be wraped and the horizontal scrolling feature must be activated.
Please note that I have used the table inside a **LinearLayout**, which is also a child of **viewPager**

**Comment by BraisGabin** [comment]
> The lines should not be wraped and the horizontal scrolling feature must be activated.
This feature is not supported for performance reasons. #70 or #7 as reference.

**Comment by iman2420** [comment]
This feature is presented in your sample code!

**Comment by BraisGabin** [comment]
No, it's not. This library never looks at the measure of their childs. It just tell the childs "your size should be X, Y". Any way, no one mantain this library. So, if you want a wrap_content table view you can do it forking the code. You create all the views in one column, measure them and take the maximum (And re-measure them with the new size).
